src/views/Home/index.view.tsx

```diff
@@ -78,33 +95,118 @@ const mission_props = {
   },
 };
 
+const sponsors = [
```

Comment:
This should follow

Reply:
https://github.com/CruzHacks/cruzhacks-2021-website/pull/122#discussion_r550619673

Reply:
I went ahead & pulled out all homepage props into JSON imports from a `props` directory in https://github.com/CruzHacks/cruzhacks-2021-website/pull/122/commits/02437f72755ddf8099f3780b27656cfe3afcba52 — now the controller code is looking super-clean